src/kakarot/instructions/environmental_information.cairo

```diff
@@ -4,6 +4,7 @@
 
 // Starkware dependencies
 
+from starkware.cairo.common.alloc import alloc
```

Comment:
```suggestion
```
Alloc is already imported below

Reply:
Ah, saw this a bit late. Commited that fix just now :)